chapter_16_javascript.asciidoc

```diff
@@ -24,13 +24,16 @@ once the user started fixing the problem?
 Just like our nice HTML5 validation errors do?
 For that we'd need a teeny-tiny bit of JavaScript.
 
+
+// RITA: In this paragraph, please tell us how we're going to play around with JavaScript and what it'll teach us. I mean, you're going to use Javascript to make it so the duplicate item error message disappears once the user fixes the problem.
 Python is a delightful language to program in.
 JavaScript wasn't always that.
 But many of the rough edges have been smoothed off,
-and I think it's fair to say that it's actually quite nice now.
-And in the world of web development, it's unavoidable.
+and I think it's fair to say that JavaScript is actually quite nice now.
+And in the world of web development, using JavaScript is unavoidable.
 So let's dip our toes in, and see if we can't have a bit of fun.
 
+// RITA: Consider dropping mention of JavaScript: The Good Parts and just referencing your more recent recommendations. If you've read Eloquent JavaScript, consider dropping "I've heard good things" to make your recommendation sound more confident. I suppose you could still mention JavaScript: The Good Parts after that.
```

Comment:
i have not read "eloquent javascript" unfortunately, so i really don't have any better recommendation...

i'll ask the tr friends!